java-client/pom.xml

```diff
@@ -94,7 +94,7 @@
 		<dependency>
 			<groupId>org.json</groupId>
 			<artifactId>json</artifactId>
-			<version>20190722</version>
+			<version>20230227</version>
```

Comment:
AFAIR it's here only to provide a version for microservices.
Optional: could try to remove it

Reply:
Actually yes - it is only used in a single method `com.cumulocity.sdk.client.notification.MessageExchange.ResponseConsumer#retryHandleContent()`. Nothing to do with microservice version from what I gather, but just to parse a JSON array?